context/state.ts

```diff
@@ -191,33 +191,30 @@ function reducer(state: tezosState, action: action): tezosState {
       };
     }
     case "removeContract": {
-      let { [action.address]: _, ...contracts } = state.contracts;
```

Comment:
the bug seems still there (when imported 3 wallets?)

Reply:
The bug is FINALLY fixed. The way I fix it is not optimal, but on the other hand, disabling the delete for 3s is a good thing because it'll avoid missclicking and deleting a wallet that you don't want to delete